Kernel/Graphics/VirtIOGPU/VirtIOGraphicsAdapter.h

```diff
@@ -38,5 +40,4 @@ class VirtIOGraphicsAdapter final : public GraphicsDevice {
     RefPtr<VirtIOGPU> m_gpu_device;
     bool m_created_framebuffer_devices { false };
 };
-
 }
```

Comment:
For the sake of keeping symmetry with the empty line after `namespace Kernel::Graphics {` (L13), let's not remove this empty line :P